pkg/common/DnnImageFeaturizer.targets

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <Content Include="$(MSBuildThisFileDirectory)..\..\tools\DnnImageFeaturizer\**\*.*">
```

Comment:
Why is this folder named `DnnImageFeaturizer`, but the Link uses `DnnImageModels`? I think they should be consistent.

Reply:
Renamed for consistency.

---
In reply to: [236760874](https://github.com/dotnet/machinelearning/pull/1447#discussion_r236760874) [](ancestors = 236760874)